test/integration/targets/async/tasks/main.yml

```
@@ -124,6 +127,7 @@
     that:
     - async_result.ansible_job_id is match('j\d+\.\d+')
     - async_result.finished == 1
+    - async_result.finished
     - async_result is finished
     - async_result is not changed
     - async_result is failed
```

This can be simplified:

```suggestion
    - async_result.finished
    - async_result.changed is false
    - async_result.failed
```